prinz_pmml/src/main/scala/pl/touk/nussknacker/prinz/pmml/model/PMMLModelInstance.scala

```diff
@@ -1,11 +1,48 @@
 package pl.touk.nussknacker.prinz.pmml.model
 
-import org.jpmml.evaluator.Evaluator
-import pl.touk.nussknacker.prinz.model.ModelInstance
+import org.dmg.pmml.FieldName
+import org.jpmml.evaluator.{Evaluator, EvaluatorUtil, FieldValue, PMMLException}
+import pl.touk.nussknacker.engine.util.SynchronousExecutionContext.ctx
+import pl.touk.nussknacker.prinz.model.{ModelInstance, ModelRunException}
+import pl.touk.nussknacker.prinz.pmml.model.VectorMultimapUtils.VectorMultimapAsRowset
 import pl.touk.nussknacker.prinz.util.collection.immutable.VectorMultimap
 
+import java.util.{Map => JMap}
+import scala.jdk.CollectionConverters.{mapAsJavaMapConverter, mapAsScalaMapConverter}
+import scala.concurrent.Future
+
 case class PMMLModelInstance(evaluator: Evaluator, override val model: PMMLModel)
   extends ModelInstance(model, PMMLSignatureProvider) {
 
-  override def run(inputMap: VectorMultimap[String, AnyRef]): ModelRunResult = ???
+  type PMMLArgs = JMap[FieldName, FieldValue]
+
+  override def run(inputMap: VectorMultimap[String, AnyRef]): ModelRunResult = Future {
```

Comment:
can you rebase this branch on the master branch as there the VectorMultimap[String, AnyRef] has type defiinition ( type ModelInputData = VectorMultimap[String, AnyRef]) which should be used here

Reply:
I just noticed that this would make a full merge of the main branch into `dev/pmml`. Can we make a separate change to merge main into `dev/pmml` (or even better, rebase `dev/pmml` on `master`) and then merge this PR?

Leaving unresolved for now. I dropped cherry-picks from master since this PR had 60+ files and you wouldn't be able to tell my changes and the main branch apart.

Reply:
Rebased `dev/pmml` and fixed the comment.